en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md

```diff
@@ -6,9 +6,11 @@ iconBrand: react
 
 # Introduction to the "React" layer
 
-Some admin modules render their UI with React, a popular Javascript library created by Facebook. 
-For these sections, rendering happens via client side scripts that create and inject HTML 
-declaratively using data structures. 
+Some admin modules render their UI with React, a popular Javascript library created by Facebook.
+For these sections, rendering happens via client side scripts that create and inject HTML
+declaratively using data structures.
+
+Even within sections that are _not_ primarily rendered in react, several React components may be injected into the DOM.
```

Comment:
```suggestion
Some admin modules render their UI with React, a popular Javascript library created by Facebook. For these sections, rendering happens via client side scripts that create and inject HTML declaratively using data structures.

Even within sections that are _not_ primarily rendered in react, several React components may be injected into the DOM.
```

Reply:
I don't understand the need for this change, but it won't change anything so I'll do it lol

Reply:
https://github.com/silverstripe/developer-docs/pull/130#discussion_r1054846663